src/lib/pci_virtio_block.c

```diff
@@ -340,8 +396,16 @@ pci_vtblk_init(struct pci_devinst *pi, char *opts)
 
 	pthread_mutex_init(&sc->vsc_mtx, NULL);
 
+	/* Customise the capabilities per-device */
+	vi_consts = (struct virtio_consts*)malloc(sizeof(struct virtio_consts));
```

Comment:
not testing for malloc failure here?

Reply:
Ooh yes good point. I've added and force-pushed a NULL check on the next line ⬇️

Reply:
Looks good. There is a calloc a few lines above also unchecked. But that’s unrelated to this PR